components/02-elements/rating/rating--summary.hbs

```diff
@@ -1,5 +1,5 @@
-<div class="rating__summary {{ class }}" {{{ attributes }}}>
-    <span class="rating__fill" {{{ percentFilled }}}>
-        {{{ percent }}}
-    </span>
+<div class="rating rating--summary {{ class }}" {{{ attributes }}}>
+    <div class="rating__star {{ star.class }}" {{{ star.attributes }}}>
+        <span class="rating__indicator {{ indicator.class }}" {{{ indicator.attributes }}}></span>
+    </div>
```

Comment:
I'm wondering - it looks like we have `--rate` variant and modifier and not `--summary` Maybe will be better to refactor it to be more readable?